pro/src/screens/CollectiveOfferSummaryEdition/__specs__/CollectiveOfferSummaryEdition.spec.tsx

```diff
@@ -122,4 +123,54 @@ describe('CollectiveOfferSummary', () => {
       }
     )
   })
+
+  it('should display desactive offer when clicking on activate offer', async () => {
```

Comment:
Tu pourrais simplifier l'intitulé, par exemple `should activate offer`, et pour l'autre `should deactivate offer` - et il me semble qu'un seul des deux tests suffirait, ils testent le même comportement

Reply:
les changements sont disponibles, c'est plus propre effectivement